src/stdLambda.cls

```diff
@@ -738,7 +759,7 @@ Private Sub parseFlowPriority1()
         this.stackSize = size
         
         If optConsume("end") Then
-            Call addOperation(oPush, , 0, 1) 'Expressions should always return a value
+            ' Why should it always return? I disabled it for now because in a loop it keeps pushing onto the stack Call addOperation(oPush, , 0, 1) 'Expressions should always return a value

```

Comment:
Removing this from this function has me very nervous. I'm not sure what the side effects would be of removing this.

In order to remove from the stack you can add your own cleanup in the loop. Either using `iPop` or `iMerge`.